src/components/ui/UiRennesLogo.vue

```diff
@@ -0,0 +1,9 @@
+<script lang="ts" setup>
+import rennesLogo from '@/assets/icons/rennesLogo.png'
+</script>
+
+<template>
+  <div class="h-8 w-32 flex flex-col justify-center">
+    <img :src="rennesLogo" />
```

Comment:
The alt attribute is missing

Reply:
Not sure what should I put here for the alt attribute, it's not defined in Figma design.